h2/src/main/org/h2/expression/function/Function.java

```diff
@@ -1157,7 +1167,7 @@ protected static Value getNullOrValue(Session session, Expression[] args,
         return v;
     }
 
-    protected Value getValueWithArgs(Session session, Expression[] args) {
+    private Value getValueWithArgs(Session session, Expression[] args) {
```

Comment:
If you can't see CI logs for some reason, the problem is here. This method needs to have `protected` visibility, because it is overridden in mode-specific subclass.

Reply:
thanks, doing this by remoting in to my work machine since I'm on holiday :-)